README.md

```diff
@@ -85,6 +85,8 @@ Once you've fetched the player, display the player in the DOM. You can re-use co
 **YOUR NOTES**
 ```
 
+
+I first rendered the player to the page using innerHTML but then was unable to isolate the button / likes elements on the page (they were returning null) so had to redo creating the individual attributes. Is there a way to do it with innerHTML?
```

Comment:
When you're using `.innerHTML` and need to attach an event listener to one of the elements you're creating, you would need to use a `querySelector` after calling `.innerHTML`. Something like this:

```js
div.innerHTML = `
<button class="delete">Delete</button>
`

// querySelector finds the element within the div
const deleteBtn = div.querySelector(".delete")
deleteBtn.addEventListener("click", handleDeleteClick)
```